omnij-core/src/main/java/foundation/omni/tx/EncodeMultisig.java

```diff
@@ -21,7 +21,7 @@
 
     private static final int maxKeys = 3;  /* Redeemable key + 2 data keys */
     private static final int maxDataKeys = maxKeys - 1;
-    private static final int dustAmountInSatoshis = 100000;
+    private static final int dustAmountInSatoshis = 684;
```

Comment:
If there are more than two public keys (i.e. three including the sender's key), or if the sender's public key is not compressed, then this value is too low, assuming default values for `-minrelayfee`. A list of default values can be found here: https://github.com/mastercoin-MSC/mastercore/commit/426592e287f79b37fbdddb5cad14ce9bf2ca1317

Bitcoin and Omni Core produce compressed public keys via `getnewaddress`, so the remaining question is: do the payloads that are embedded via `encode` fit into one public key, or do they require two?